src/components/security-rules/configure-rule/ConfigureRule.jsx

```diff
@@ -488,12 +489,15 @@ const ConfigureRule = (props) => {
         </ConditionalFormBlock>
         <ConditionalFormBlock
           dependency="rule"
-          condition={() => form.getFieldValue('rule') === "webhook"}
-        >
+          condition={() => form.getFieldValue('rule') === "webhook"} >
           <FormItemLabel name="URL" />
           <FormItem name="url" rules={[{ required: true }]}>
             <Input placeholder="URL" />
           </FormItem>
+          <FormItemLabel name="STORE" />
+          <FormItem name="store" rules={[{ required: false }]}>
+              <Input placeholder="The variable in which the webhook response is stored. For example: args.res" />
```

Comment:
Let's rephrase the placeholder to this - `The variable to store the webhook response. For example: args.res`. Also update the placeholder in the `query`'s `store` with a similar structure.